Packs/Automox/Integrations/Automox/Automox.py

```diff
@@ -831,7 +831,7 @@ def update_group(client: Client, args: Dict[str, Any]) -> CommandResults:
 
 def upload_vulnerability_sync_file(client: Client, args: Dict[str, Any]) -> CommandResults:
     org_id = args.get(ORG_IDENTIFIER, None) or DEFAULT_ORG_ID
-    report_source = args.get('report_source', None)
+    report_source = args.get('reports_source', None)
```

Comment:
Just to confirm, we are keeping this plural because if we don't it's a major version bump right?